signalr-node-server/signalr-http.js

```diff
@@ -55,35 +58,41 @@ class HttpTransport {
         this._httpServer.on('request', (req, res) => {
             var parsedUrl = url.parse(req.url, true);
 
+            // Set CORS headers
```

Comment:
Added CORS headers, maybe make configurable?

Reply:
Isn't there a CORS middleware as part of express? I don't want to bake CORS headers into the library.

Reply:
I'll remove it from the PR. I did like the example for the .NET version on how to add CORS. Maybe there should be a seperate example (wiki page even here on Github?)